scripts/unpack

```diff
@@ -103,9 +108,55 @@ if [ -d "$SOURCES/$1" -o -d "$PKG_DIR/sources" -o -n "$PKG_GIT_URL" ]; then
   fi
 
   if [ -z "$PKG_URL" -a -n "$PKG_GIT_URL" ]; then
-    rm -rf $BUILD/$PKG_NAME-$PKG_VERSION
+    # init GIT_ORIG_DIR
+    GIT_ORIG_DIR=""
+    # cycle through folders that match $PKG_NAME in $BUILD until we find the right one
+    for f in $BUILD/$1-* ; do
+      echo "**** debug: checking folder '$f':"
+      if [ -d $f ]; then
+        echo "**** debug: '$f' is folder"
+        if [ -f $f/.libreelec-unpack ]; then
+          echo "**** debug: '.libreelec-unpack' found"
+          source $f/.libreelec-unpack
+          echo "**** debug: STAMP_PKG_NAME=$STAMP_PKG_NAME"
+          if [ "$PKG_NAME" = "$STAMP_PKG_NAME" ]; then
+            echo "**** debug: NAME matched"
+            if [ -f $f/.git/config ]; then
+              echo "**** debug: .git/config found:"
+              cat $f/.git/config
+              GIT_MATCH=$(grep "$PKG_GIT_URL" $f/.git/config | wc -l)
```

Comment:
Rather than parsing .git/config, you could likely use `git remote get-url origin`